starlite_users/route_handlers.py

```diff
@@ -1,9 +1,9 @@
-from typing import TYPE_CHECKING, Any, Callable, Dict, List, Literal, Optional, Type, Union
+from __future__ import annotations
```

Comment:
I've imported this previously in this module and it caused tests to break on Python 3.8
Maybe this is different in v2, I'll give it a shot.

Reply:
I'm actively working to make it a non-issue - so if you have any LMK.

On every layer you can define a `signature_namespace` mapping that maps names to types which we use to resolve the annotations in the handler, and if you are using starlite special kwarg names (think State, Scope, Receive, Send etc etc, those are always included in the namespace that we use to resolve the forward referenced parameter types, so they don't need to be imported explicitly outside of if TYPE_CHECKING anymore, and should support future annoations.